src/theory/quantifiers/quant_conflict_find.h

```diff
@@ -309,6 +248,63 @@ class QuantConflictFind : public QuantifiersModule
    * algorithm performed by this class.
    */
   bool isPropagatingInstance(Node n) const;
+
+  enum Effort : unsigned
```

Comment:
The indentation is off because of the indentation of `public:` above. Can you try indenting `public:` by one space and reapplying ClangFormat?